Collaboration.md

```diff
@@ -0,0 +1,155 @@
+# Bitcoin Design Community collaboration playbook
+
+As we get actively involved in supporting projects, we will probably encounter a few different types of situations. The goal of this document is to define common types of collaborations, to serve as starting points for new inquiries.
+
+This is not meant to be a strict rule set. Every project is unique, and healthy collaboration requires everyone involved to be sensitive to the projects needs and goals. Our own goal is to support projects and contributors to improve their products and design practices (and not impose our ideas). 
+
+Here are six types of design needs a project may have.
+
+## #1 Resources & consultation
+
+An existing project is looking for resources and expertise to improve their product's user experience (and potentially design process).
+
+Ways the project can choose to interact with the community:
+
+1. Review the guide to find a direction or solution
+1. Engage in Slack conversations to share ideas and get feedback
+1. Participate in a design review session
```

Comment:
I think these are a bit generic steps, lets' try to give a concrete example. I like the[ analysis approach](https://github.com/BitcoinDesign/Guide/discussions/101#discussioncomment-294594) Steve recommended here

Reply:
These steps are written from the perspective of a project that wants to reach out - what others are probably going to do. And then we can respond to that in whatever way we think is appropriate based on the project and their needs.

What methods and tools we use to respond to inquiries is a bigger topic, as we can borrow an endless number of techniques from the fields of user research and interaction design, like [all the stuff here](https://www.designkit.org/methods). Didn't want to go too deep into techniques, as different designers prefer different ones also.

Those were my thoughts when writing this. If it's not clear or useful, the let's change it.

Reply:
I misunderstood who's addressing here. Now that you mentioned it is okay (from the project perspective) I understood it's from a designer perspective (our).